aiqc/__init__.py

```diff
@@ -3199,9 +3186,10 @@ def transform_dynamicDimensions(
 			# This `.T` works for both single and multi column.
 			encoded_samples = samples_to_transform.T
 			# Since each column is 1D, we care about rows now.
+			
 			length = encoded_samples.shape[0]
 			if (length == 1):
-				encoded_samples = fitted_encoders[0].transform(encoded_samples)
+				encoded_samples = fitted_encoders[0].transform(encoded_samples[0])
```

Comment:
please add some comments here. as i understand it a really specific bandaid is being made here.